modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java

```diff
@@ -62,15 +61,14 @@ public void portalInstanceRegistered(Company company) throws Exception {
 		Collection<Long> classNameIds =
 			_contentDashboardItemFactoryTracker.getClassNameIds();
```

Comment:
@guilhermedcamacho the ContentDashboardItemFactoryTracker has a "getClassNames" method. Just use that so it does not need to covert className and classId back and forth.